lib/ramble/ramble/experiment_result.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022-2024 The Ramble Authors
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+
+_DICT_MAPPING = {
+    "name": "name",
```

Comment:
Is it possible to use the namespace attributes where they make sense here?

Reply:
Definitely, that's a great suggestion, done.